widgets/contact/tpl/default.php

```diff
@@ -35,25 +42,13 @@
 		<input type="hidden" name="instance_hash" value="<?php echo esc_attr($instance_hash) ?>" />
 
 		<?php if( $use_recaptcha ) : ?>
-			<div class="g-recaptcha" data-sitekey="<?php echo esc_attr( $instance['spam']['recaptcha']['site_key'] ) ?>" data-callback="soOnCaptchaSuccess"></div>
-			<?php if( !is_admin() ) : ?>
-			<script type="application/javascript">
-				jQuery(function ($) {
-					// Ensure we're getting the right submit input in case there are multiple widgets on a page.
-					var $input = $("<?php echo '.js-sow-submit-' . $instance['_sow_form_id'] ?>");
-					window.soOnCaptchaSuccess = function (response) {
-						if(response) {
-							$input.prop('disabled', false);
-						}
-					};
-				});
-			</script>
-			<?php endif; ?>
+			<div class="sow-recaptcha"
+				 data-config="<?php echo esc_attr( json_encode( $settings ) ) ?>"></div>
 		<?php endif; ?>
 
 		<div class="sow-submit-wrapper <?php if( $instance['design']['submit']['styled'] ) echo 'sow-submit-styled' ?>">
 			<input type="submit" value="<?php echo esc_attr( $instance['settings']['submit_text'] ) ?>"
-				   class="sow-submit <?php echo 'js-sow-submit-' . $instance['_sow_form_id'] ?>" <?php if( $use_recaptcha && !is_admin() ) echo 'disabled="true"'; ?>>
+				   class="sow-submit" <?php if( $use_recaptcha && !is_admin() ) echo 'disabled="true"'; ?>>
```

Comment:
Nope. I don't think many sites will work with JS disabled these days. Besides the fact that the Recaptcha API is loaded as a JS file. :grin: